### PR TITLE
[resolves #2392] project.build.sourceEncoding Maven property not set

### LINKED ISSUE #2392: project.build.sourceEncoding Maven property not set
```
[WARNING] Using platform encoding (UTF-8 actually) to copy filtered resources, i.e. build is platform dependent!
```
We should set it to `utf-8`.

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -126,6 +126,9 @@
         <!-- GitHub -->
         <github.namespace>wildfly-extras</github.namespace>
         <github.project>wildfly-camel</github.project>
+
+        <!-- Common Maven properties -->
+        <project.build.sourceEncoding>utf-8</project.build.sourceEncoding>
```
Comment (jamesnetherton) [inline pom.xml]: Nitpicking a bit here, but can we go with uppercase `UTF-8` for consistency.

Comment (ppalaga) [reply pom.xml]: "for consistency" with what?

Comment (jamesnetherton) [reply pom.xml]: Consistency with:

* How it's written in the rest of the project
* The example in official maven docs https://maven.apache.org/general.html
* Every other [maven project](https://github.com/search?l=Maven+POM&p=1&q=sourceEncoding&type=Code&utf8=%E2%9C%93) on the planet :wink:

**Comment by ppalaga** [comment]
In ef57dd70b9fb51bea9addaacb2352f08aae94cae `utf-8` is replaced with `UTF-8` as proposed by @jamesnetherton

**Comment by jamesnetherton** [comment]
ok to test

**Pull request merged**